kahuna/public/js/search/results.js

```diff
@@ -350,6 +352,15 @@ results.controller('SearchResultsCtrl', [
             selection.clear();
         };
 
+        ctrl.shareSelection = async () => {
+            $q.all(ctrl.selectedImages.map(image => { return image.data.id; }).toArray()).
+            then(sharedImagesIds => {
+              const sharedUrl = $window._clientConfig.rootUri + "/search?nonFree=true&ids=" + sharedImagesIds.join(',');
+              navigator.clipboard.writeText(sharedUrl);
+              globalErrors.trigger('clipboard', sharedUrl);
+            });
+        };
```

Comment:
why is this a promise? I can't see anything async here
```suggestion
        ctrl.shareSelection = () => {
            const sharedImagesIds = ctrl.selectedImages.map(image => image.data.id);
            const sharedUrl = $window._clientConfig.rootUri + "/search?nonFree=true&ids=" + sharedImagesIds.join(',');
            navigator.clipboard.writeText(sharedUrl);
            globalErrors.trigger('clipboard', sharedUrl);
        };
```